doc/news/changes/incompatibilities/20170910DavidWells

```diff
@@ -0,0 +1,20 @@
+Changed: The methods
+<ol>
+  <li>Manifold::get_new_point</li>
+  <li>Manifold::project_to_manifold</li>
+</ol>
+have had their declarations changed in an incompatible manner: these three
+methods now take arguments that are ArrayView​s instead of
```

Comment:
well, it's actually only two methods :-)

If you write it as "ArrayView objects", then doxygen will link to the class correctly.